kiva/tests/test_basecore2d.py

```diff
@@ -111,7 +111,7 @@ def test_get_ctm(self):
         # default ctm should be identity matrix.
         desired = affine.affine_identity()
         actual = gc.get_ctm()
-        self.assertTrue(alltrue(ravel(actual == desired)))
+        self.assertTrue(all(ravel(actual == desired)))
```

Comment:
If you really want to clean things up, all of these `self.assertTrue(all(... == ...))` can be replaced with:
```
from numpy.testing import assert_array_equal
...
assert_array_equal(..., ...)
```
eg. the above could be simply `assert_array_equal(actual, desired)`